docs/schemas/README.md

```diff
@@ -0,0 +1,17 @@
+# Configuration Schemas
```

Comment:
Is your plan to update https://github.com/Azure/AppConfiguration/tree/main/docs/FeatureManagement pointing to here?

Reply:
It wasn't. Do you see some value in that?

Reply:
Never mind. That's the schema for AppConfig, which needs a new schema for dynamic feature support of its own. No need to link here.